app.py

```diff
@@ -33,36 +33,65 @@ def videogame_POST():
-            videogame = VideoGame(data["title"], data["platform"])
+            # Save title and platform dict values into videogame variable. Constructor creates an empty arg. 
+            videogame = Videogame()
+            videogame.videogame_title = data["title"]
+            videogame.videogame_platform = data["platform"]
+            videogame.videogame_releasedate = data["releasedate"]
+            videogame.videogame_publisher = data["publisher"]
+
+            videogame.videogame_id = str(uuid.uuid4())
+            db.session.add(videogame)
+            db.session.commit()
 
             # Generate and pass uuid property for each new videogame entry
-            data["uuid"] = uuid.uuid4()
+            data["uuid"] = videogame.videogame_id
             data["resource-path"] = f'/videogame/{data["uuid"]}'
 
+            # SQLAlchemy. Review db.session.commit() func
+
             # Generate successful response for submission
-            return jsonify(data), 201
+            return jsonify(data), 201 # Created
 
         if "title" not in data.keys():
             error_message["message"] = "Title wasn't included in data set submission"
-            return jsonify(error_message), 400
+            return jsonify(error_message), 400 # Bad request
 
         elif "platform" not in data.keys():
             error_message["message"] = "Platform wasn't included in data set submission"
-            return jsonify(error_message), 400
+            return jsonify(error_message), 400 # Bad request
 
         else:
             return 'Request not being understood. Check previous if and elif statements'
 
-    # Successful POST: curl -X POST http://127.0.0.1:5000/videogames -H "Content-Type: application/json" -d '{"title":"Spiderman 2", "platform":"PlayStation"}'
-    # Inaccurate JSON file: curl -X POST http://127.0.0.1:5000/videogames -H "Content-Type: application/json" -d '{"name":"Spiderman 2", "platform":"PlayStation 2"}'
-
 
-@app.route('/videogames/<uuid>', methods=['GET', 'PATCH', 'DELETE'])
-def unique_info(uuid):
-    assert resource == request.view_args['uuid']
-    return "Pull specific info on video games in database."
+@app.route('/videogames/<uuid>', methods=['GET'])
+# GET Request Example: curl -X GET http://127.0.0.1:5000/videogames/88cb6cae-f55b-4f65-839c-ab58a5d88e91
+# Query for UUID entry in the Videogame Database
+def info_query(uuid):
+    entry_info = db.session.query(Videogame).filter(Videogame.videogame_id == uuid).first()
+    if entry_info:
+        # print(f"The requested ID is: {entry_info.videogame_id}\nThe title is: {entry_info.videogame_title}\nThe platform is: {entry_info.videogame_platform}")
+        videogame_query = "The requested ID is: {entry_info.videogame_id}\nThe title is: {entry_info.videogame_title}\nThe platform is: {entry_info.videogame_platform}\nThe release date is: {entry_info.videogame_releasedate}\nThe publisher is: {entry_info.videogame_publisher}"
```

Comment:
You don't need all the other data. This should look like the information that comes back from POST, just the json body with the video game info.